yt_dlp/YoutubeDL.py

```diff
@@ -3792,6 +3789,15 @@ def get_encoding(stream):
                 proxy_map.update(handler.proxies)
         write_debug(f'Proxy map: {proxy_map}')
 
+        for plugin_type, plugins in {'Extractor': plugin_extractors, 'Post-Processor': plugin_postprocessors}.items():
+            if not plugins:
+                continue
+            write_debug(f'{plugin_type} Plugins: %s' % (', '.join(sorted(('%s%s' % (
+                klass.__name__, '' if klass.__name__ == name else f' as {name}')
+                for name, klass in plugins.items())))))
+        plugin_dirs = plugin_directories()
+        if plugin_dirs:
+            write_debug('Plugin directories: %s' % plugin_directories())
```

Comment:
```suggestion
            write_debug('Plugin directories: %s' % plugin_dirs)
```

use already created variable rather invoking again

Reply:
If we change the line anyways why not also start using fstrings?
```suggestion
            write_debug(f'Plugin directories: {plugin_dirs}')
```